app/views/mainstream_browse_pages/_create_update_form.html.erb

```diff
@@ -0,0 +1,60 @@
+<%= form_for @browse_page do |f| %>
+  <% unless update %>
```

Comment:
I always find `unless` conditional logic a bit of a mind bender. I wonder if this would benefit from a comment just to explain it in terms that my brain understands.

```suggestion
  <%# Only show parent selection when creating a new page %>
  <% unless update %>
```